server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -1355,6 +1355,126 @@ public void setMessagesShouldAllowDraftCreation() {
             .body(ARGUMENTS + ".created", hasKey(messageCreationId));
     }
 
+    @Test
+    public void setMessagesShouldCreateDraftOutsideOfDraftMailbox() {
```

Comment:
What is the difference between this test and "setMessagesShouldAllowDraftCreationOutsideOfDraftMailbox"

Reply:
Well catched, it's duplicated, I removed it.